### PR TITLE
add skip button on contact detail page

### PR DESCRIPTION
**Pull request by DianeLiu2019**
# Description

add skip button and skip information on contact detail page. 
Fixes #1173(issue)

> Please delete options that are not relevant.

- [ x] New feature

# Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made any needed changes to the documentation
- [ ] My changes generate no new warnings
- [x ] I have added tests that prove my fix is effective or that my feature works
- [ x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: f2/src/forms/ContactInfoForm.js [hunk 1]
```diff
@@ -60,11 +64,29 @@ export const ContactInfoForm = ({ onSubmit }) => {
...
+          
+          <Lead>
+          <Trans id="contactinfoPage.skipInfo"/>
+          </Lead>
+          <Button
+        as={ReactRouterLink}
+        fontSize={{ base: 'lg', md: 'xl' }}
+        color="black"
+        variant="link"
+        variantColor="teal"
```
Comment (amazingphilippe) [inline f2/src/forms/ContactInfoForm.js]: variantColor should be gray

File: f2/src/forms/ContactInfoForm.js [hunk 2]
```diff
@@ -60,11 +64,29 @@ export const ContactInfoForm = ({ onSubmit }) => {
             next={<Trans id="contactinfoPage.nextInfo" />}
             button={<Trans id="contactinfoPage.nextButton" />}
           />
+          
+          <Lead>
```
Comment (amazingphilippe) [inline f2/src/forms/ContactInfoForm.js]: This should be a P instead of a Lead

File: f2/src/locales/fr.json [hunk 3]
```diff
@@ -73,6 +73,8 @@
   "contactinfoPage.nextInfo": "Ensuite, on vous demandera d'examiner votre rapport.",
   "contactinfoPage.or": "Ou",
   "contactinfoPage.phoneNumber": "Inscrivez votre numéro de téléphone.",
+  "contactinfoPage.skipButton": "sauter",
```
Comment (amazingphilippe) [inline f2/src/locales/fr.json]: Capital S

File: f2/src/forms/ContactInfoForm.js [hunk 4]
```diff
@@ -63,17 +62,17 @@ export const ContactInfoForm = ({ onSubmit }) => {
           <NextAndCancelButtons
             next={<Trans id="contactinfoPage.nextInfo" />}
             button={<Trans id="contactinfoPage.nextButton" />}
-          />
-          
-          <Lead>
+          />        
+          <p>
```
Comment (amazingphilippe) [inline f2/src/forms/ContactInfoForm.js]: use the P component (capital P)

File: f2/src/forms/ContactInfoForm.js [hunk 5]
```diff
@@ -63,17 +62,17 @@ export const ContactInfoForm = ({ onSubmit }) => {
...
+          </p>
           <Button
         as={ReactRouterLink}
         fontSize={{ base: 'lg', md: 'xl' }}
         color="black"
-        variant="link"
-        variantColor="teal"
+        // variant="link"
+        variantColor="gray"
+        bg = "gray.400"
```
Comment (amazingphilippe) [inline f2/src/forms/ContactInfoForm.js]: prettify markup and remove comment!

File: f2/src/forms/ContactInfoForm.js (hunk 4 above)
Comment (DianeLiu2019) [reply f2/src/forms/ContactInfoForm.js]: fixed

File: f2/src/forms/ContactInfoForm.js (hunk 5 above)
Comment (DianeLiu2019) [reply f2/src/forms/ContactInfoForm.js]: Thanks

File: f2/src/locales/fr.json (hunk 3 above)
Comment (DianeLiu2019) [reply f2/src/locales/fr.json]: Thanks

File: f2/src/forms/ContactInfoForm.js (hunk 1 above)
Comment (DianeLiu2019) [reply f2/src/forms/ContactInfoForm.js]: Thanks

File: f2/src/locales/fr.json (hunk 3 above)
Comment (DianeLiu2019) [reply f2/src/locales/fr.json]: done

File: f2/src/forms/ContactInfoForm.js (hunk 1 above)
Comment (DianeLiu2019) [reply f2/src/forms/ContactInfoForm.js]: done

File: f2/src/forms/ContactInfoForm.js (hunk 2 above)
Comment (DianeLiu2019) [reply f2/src/forms/ContactInfoForm.js]: ringht,fixed

**Pull request merged**